Add description about status and types (#1)
aergoio/aips: AIPS/aip-1.md

# What is an AIP?

AIP stands for Aergo Improvement Proposal. An AIP is a design document providing information to the Aergo community, or describing a new feature for Aergo or its processes or environment. The AIP should provide a concise technical specification of the feature and a rationale for the feature. The AIP author is responsible for building consensus within the community and documenting dissenting opinions.

> # AIP Status ( Work Flow )

Reviewer: ```suggestion
# AIP Status (Work Flow)
```

> # AIP Status ( Work Flow )

Reviewer: ```suggestion
# AIP Status (Work Flow)
```

> # AIP Status ( Work Flow )

Reviewer: ```suggestion
# AIP Status (Work Flow)
```

> Basically anyone can suggest their ideas to Aergo community.

Reviewer: ```suggestion
Everyone can suggest their ideas to the Aergo community.
```

>  1. None

Reviewer: ```suggestion
 1. Start
```

     * Write improvement idea and create a pull request on Github (https://github.com/aergoio/aips).
 2. Active

>      * Aergo team will review the pull request and discuss on the GitHub.

Reviewer: ```suggestion
     * Aergo team will review the pull request and discuss on GitHub.
```

 3. Develop

>      * If idea is accepted, Aergo Team starts to develop.

Reviewer: ```suggestion
     * If idea is accepted, Aergo team starts to develop.
```

>      * If idea is accepted. Aergo Team start to develop.

Reviewer: ```suggestion
     * If idea is accepted, Aergo Team starts to develop.
```

 4. Applied

>      * It will be applied on the mainnet by code.

Reviewer: ```suggestion
     * After the code is released and tested, the mainnet will be upgraded.
```

# AIP Type

> There are n types of AIP:

Reviewer: ```suggestion
There are 3 types of AIP:
```

 1. Standard
     * Describes any change that affects most or all Aergo implementations.
 2. Parameter

>      * Describes a blockchain parameters. for example, changing the number of block producers.

Reviewer: ```suggestion
     * Describes a blockchain parameter. for example, changing the number of block producers.
```

>      * Describes a blockchain parameter. for example, changing the number of block producers.

Reviewer: ```suggestion
     * Describes the change of a blockchain parameter. For example, changing the number of block producers.
```
